huntsman/utils/pyro/event.py

```diff
@@ -0,0 +1,27 @@
+from threading import Event
+
+event_types = {"camera",
+               "focuser",
+               "filterwheel"}
+
+
+class RemoteEvent(Event):
+    """Interface for threading.Events of a remote camera or its subcomponents (e.g. filterwheel)
+    """
```

Comment:
Maybe list all the types in the docstring itself. Either in description or providing params description.
```suggestion
    """Interface for threading.Events of a remote camera or its subcomponents.
    
    Current supported types are: `camera`, `focuser`, `filterwheel`.
    """
```